test/function/samples/empty-string-as-module-name/_config.js

```diff
@@ -0,0 +1,15 @@
+module.exports = {
+	description: 'allows using empty string as a valid module name',
```

Comment:
When writing and debugging tests, an important feature is that you can add `solo: true` on the top level to just run a single test. You can also add `show: true` to always show the generated code even if the test succeeded. Both should never be committed, though.